tests/microos/disk_boot.pm

```diff
@@ -31,4 +31,14 @@ sub test_flags {
     return {fatal => 1};
 }
 
+sub post_fail_hook {
+    assert_screen 'emergency-mode';
```

Comment:
What happens if the `assert_screen` fails in the `post_fail_hook`?

Reply:
Nothing, we won't just get the initramfs output.